test-activity.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
```

Comment:
TODO: we probably don't want to commit this in the acs-engine repo, but perhaps there's an appropriate place for L&P scripts like this one.